src/granite/query/builder.cr

```diff
@@ -153,6 +153,16 @@ class Granite::Query::Builder(Model)
     assembler.count
   end
 
+  def aggregate(field : String | Symbol, t)
+    assembler.aggregate(field, t).run
+  end
```

Comment:
I'm not sure why this method is here...why not just make the method defined in the for loop below call the assembler directly?

Reply:
:man_shrugging: Might have just assumed i need it?  I'll take a look tomorrow.